scripts/add_field_for_stuck_registrations/QA_script.csv

```diff
@@ -0,0 +1,4 @@
+64786f88-03f5-4ffd-96eb-e54ae9122d21
+f80155d1-11f1-4f69-aa21-77d5907e387e
+e2770fb4-fd91-43b9-a343-74d3ee15d19d
+7f4620b3-9cc8-4209-bd68-9f0d7c243060
```

Comment:
We probably don't need to add this file of UUIDs, rather make it clear in the README what kind of input this script wants.

Reply:
okay, will remove it from the folder